src/execution/mod.rs

```diff
@@ -427,7 +428,18 @@ where
     SW: StateWriter<u64> + 'static,
 {
     let span = tracing::trace_span!("Building dataflow").entered();
+
+    let rt = tokio::runtime::Builder::new_multi_thread()
+        .enable_all()
+        .build()
+        .unwrap();
```

Comment:
We are starting to have a certain number of tokio runtimes running depending on the config (recovery, tracing, and this).
This is not a problem per se, but it could impact performances.
We'd need to profile to understand if it's a problem at all, but we could think of sharing the runtime for lower priority tasks (tracing and the webserver)